### PR TITLE
GUA-547 security.txt

### PR DESCRIPTION
**Pull request by huwd**
Rainy day clear up ticket,
We should follow best pratices of the [GDS way](https://gds-way.cloudapps.digital/standards/vulnerability-disclosure.html#vulnerability-disclosure), and also do sensible things to help the wonderful world of internet security folk help us keep our services and users safe.

### CODE REVIEW

File: src/components/redirects/redirects-routes.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import * as express from "express";
 import { Request, Response } from "express";
 import { PATH_DATA } from "../../app.constants";
 import { requiresAuthMiddleware } from "../../middleware/requires-auth-middleware";
+import { WELL_KNOWN_FILES } from "../../app.constants";
```
Comment (TaherKapasi) [inline src/components/redirects/redirects-routes.ts:5]: code smell ;-)
```suggestion
import { PATH_DATA, WELL_KNOWN_FILES } from "../../app.constants";
```

Comment (huwd) [reply src/components/redirects/redirects-routes.ts:5]: oh good spot, I'll refactor that now...
🤔 I thought the linter would be complaining about that? How'd I get past CI?

Comment (huwd) [reply src/components/redirects/redirects-routes.ts]: Great that's amended now, thanks @TaherKapasi!

I'll take a look at the lintier rules... pretty sure it used to pull me up on those!

**Review by DougsCER — APPROVED** [review]

**Review by TaherKapasi — DISMISSED** [review]

**Comment by huwd** [comment]
Turns out the `res` wasn't getting passed through right, have just refactored a bit.

**Review by TaherKapasi — APPROVED** [review]

**Pull request merged**